### ISSUE TITLE
Compatibility with fisherman's plugins (not reading fishfile)

### ISSUE THREAD

**Issue by BarbzYHOOL** [danhper/fundle#35]
In fisherman's plugins, there is a "fishfile" which list dependencies needed for the plugin.

This could be handy to have.

**Comment by danhper** [comment]
I agree it could be nice to have support for fishfile, but I do not have time to work on this right now.
If you would like to send a PR, I will be happy to merge it.
Thank you.

**Comment by BarbzYHOOL** [comment]
If I manage to do this, I'll send a PR but since I automate installation with other tools, I doubt I will need to do this and will forget :D
